### ISSUE TITLE
Request about turrets

### ISSUE THREAD

**Issue by Edgapuffle** [studio-minus/ppg-bugs-and-requests#1048]
**Feature description**
Describe the requested feature. Is it a new spawnable item? Is it a new map? A new mechanic or system? How does it work?
It is a mechanic. I just tried to use a signal converter on a turret and make it fire faster, and it worked fine when I was just firing it on my own but then I tried and it didn't work when it was automatically shooting people.
**Feature purpose**
Describe the purpose here. What made you request this feature? What is its purpose? Are you sure it doesn't already exist in some form?
The purpose is to make turrets fire more rounds faster when you hook it up to signal converters and add propagation wires to it. I am requesting this because it would be fun to watch turrets kill people faster. Its purpose is to, and I have said this many times, make the turret kill people faster with signal converters. I am sure that it doesn't exist because I have tried it and it doesn't work.
**Additional context**
I would give a screenshot but I have no clue how to do that on my computer and all of the guides I have read are useless, so, sorry that I can't

**Comment by DindinYT37** [comment]
Dude, keep it simple. You just want it to give out a propagation signal when auto-firing, right?

**Comment by Edgapuffle** [comment]
Yeah that sounds about right. I was just answering all of the questions

**Comment by Edgapuffle** [comment]
> Dude, keep it simple. You just want it to give out a propagation signal when auto-firing, right?
Yeah that sounds about right. I was just answering all of the questions

**Comment by DindinYT37** [comment]
> > Dude, keep it simple. You just want it to give out a propagation signal when auto-firing, right?
>
> Yeah that sounds about right. I was just answering all of the questions

Okay, also you uh, but your own message in the quote lol.

**Comment by Edgapuffle** [comment]
> > > Dude, keep it simple. You just want it to give out a propagation signal when auto-firing, right?
> > 
> > 
> > Yeah that sounds about right. I was just answering all of the questions
> 
> Okay, also you uh, put your own message in the quote lol.

whoops

**Comment by DindinYT37** [comment]
> > > > Dude, keep it simple. You just want it to give out a propagation signal when auto-firing, right?
> > > 
> > > 
> > > Yeah that sounds about right. I was just answering all of the questions
> > 
> > 
> > Okay, also you uh, put your own message in the quote lol.
> 
> I'm very brand new to this, I literally made an account a few minutes before I posted this

Oh okay, it's fine, just telling you since I was like "wait, where's his message".
